From 0b7f7b8e98ade7ba816c2439e35cd60d9a5c67ac Mon Sep 17 00:00:00 2001
From: Martchus <martchus@gmx.net>
Date: Sun, 18 Sep 2016 13:58:28 +0200
Subject: [PATCH 05/31] Don't add resource files to LIBS parameter

Solves an issue where the generated pkg-config
files contained invalid Libs.private references
like .obj/debug/Qt5Cored_resource_res.o
---
 qmake/generators/win32/mingw_make.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/qmake/generators/win32/mingw_make.cpp b/qmake/generators/win32/mingw_make.cpp
index 096b041056..eb498b28e8 100644
--- a/qmake/generators/win32/mingw_make.cpp
+++ b/qmake/generators/win32/mingw_make.cpp
@@ -152,7 +152,7 @@ void MingwMakefileGenerator::init()
 
     processVars();
 
-    project->values("LIBS") += project->values("RES_FILE");
+    project->values("OBJECTS") += project->values("RES_FILE");
 
     if (project->isActiveConfig("dll")) {
         QString destDir = "";
-- 
2.24.1

